main.css

```diff
@@ -0,0 +1,83 @@
+
+ 
+body{
+font-family: 'Merriweather', serif;
+margin: 0;
+
+}
+.main-nav {
+    padding: 20px;
+    font-size:20px;
+    border: solid black 1px;
+    background-color: lightseagreen;
+ 
+  
+}
+
+.nav-link {
+    padding-right: 20px;
+    color: black;
+    text-decoration: none;
+   
+}
+
+
+.nav-link:last-of-type {
+    padding-right: 0;
+    width: 30px;
+  
+}
+.title-container{
+    background-color: lightcyan;
+    text-align: center;
+    padding: 10px;
+    font-family: 'Pacifico', cursive;
+}
+
+.all-posts-container {
```

Comment:
these styles aren't being applied b/c `all-posts-container` is the ID, when you want the class of that element for styling instead